drake/doc/developers.rst

```diff
@@ -43,6 +43,47 @@ available in the superbuild.
 class method runs as a unit test, and fails if it times out or calls
 ``error``. For more information, see ``drake/util/unitTest.m``.
 
+Supported Configurations
+========================
+
+The following table shows the configurations and platforms that Drake
+officially supports. Supported configurations are tested in continuous
+integration. All other configurations are provided on a best-effort basis.
+
++------------------------+-----------------+-------------------+---------+
+| Operating System       | Compiler        | Superbuild Deps   | Build   |
++========================+=================+===================+=========+
+| Ubuntu 14.04 LTS       | GCC 4.9         | Default           | Debug   |
+|                        |                 |                   +---------+
+|                        |                 |                   | Release |
+|                        |                 +-------------------+---------+
+|                        |                 | Default + MATLAB  | Release |
+|                        +-----------------+-------------------+---------+
+|                        | Clang 3.7       | Default           | Debug   |
+|                        |                 |                   +---------+
+|                        |                 |                   | Release |
++------------------------+-----------------+-------------------+---------+
+| Windows Server 2012 R2 | MSVC 14 32-bit  | Default           | Debug   |
+|                        |                 |                   +---------+
+|                        |                 |                   | Release |
+|                        +-----------------+-------------------+---------+
+|                        | MSVC 14 64-bit  | Default           | Debug   |
+|                        |                 |                   +---------+
+|                        |                 |                   | Release |
++------------------------+-----------------+-------------------+---------+
+| OS X 10.10             | Apple Clang 7.0 | Default           | Debug   |
+|                        |                 |                   +---------+
+|                        |                 |                   | Release |
++------------------------+-----------------+-------------------+---------+
+
+Official support for MATLAB on Windows and OS X is planned for 2016 Q2.
+Official support for a broader range of superbuild configurations is planned
```

Comment:
I agree.  I think a more general "we are working on adding more supported configurations to the list" statement would be more durable.